packages/atomic/src/components/common/result-list/result-list.pcss

```diff
@@ -1,257 +1,4 @@
-
-.list-wrapper.display-table {
-  display: grid;
-  overflow-x: auto;
-}
-
-.list-root.display-table {
-  @mixin atomic-result-table;
-}
-
-.list-root.display-list {
-  display: flex;
-  flex-direction: column;
-
-  atomic-result,
-  atomic-result-placeholder {
-    width: auto;
-  }
-
-  @screen desktop-only {
-    @mixin atomic-list-with-dividers;
-  }
-
-  @screen mobile-only {
-    &.image-large {
-      @mixin atomic-list-with-dividers;
-      display: grid;
-      justify-content: space-evenly;
-      grid-template-columns: minmax(auto, 35rem);
-    }
-
-    &.image-small,
-    &.image-icon,
-    &.image-none {
-      grid-row-gap: 1rem;
-
-      @mixin atomic-list-with-cards;
-    }
-  }
-}
-
-[part~='outline'][part~='result-list-grid-clickable-container'] {
-  position: relative;
-
-  @screen desktop-only {
-    & {
-      border: 1px solid transparent;
-      padding: 1rem;
-      border-radius: 1rem;
-      transition: all 0.12s ease-out;
-      &:hover {
-        border: 1px solid var(--atomic-neutral);
-        box-shadow: 0px 10px 25px var(--atomic-neutral);
-      }
-    }
-  }
-}
-
-[part='result-list-grid-clickable'] {
-  content: '';
-  position: absolute;
-  left: 0;
-  top: 0;
-  right: 0;
-  bottom: 0;
-}
-
-.list-root.display-grid {
-  display: grid;
-  justify-content: space-evenly;
-
-  atomic-result {
-    height: 100%;
-    box-sizing: border-box;
-  }
-
-  @screen desktop-only {
-    grid-template-columns: repeat(auto-fit, minmax(17rem, 1fr));
-
-    &.density-comfortable {
-      grid-row-gap: 4rem;
-      grid-column-gap: 1.5rem;
-    }
-
-    &.density-normal,
-    &.density-compact {
-      grid-row-gap: 3rem;
-      grid-column-gap: 1.5rem;
-    }
-
-    &.image-large {
-      grid-template-columns: repeat(auto-fill, minmax(19rem, 1fr));
-    }
-
-    &.image-small {
-      @media not all and (min-width: 1024px) {
-        grid-template-columns: repeat(3, 1fr);
-      }
-      @media (min-width: 1024px) {
-        grid-template-columns: repeat(4, 1fr);
-      }
-    }
-  }
-
-  @define-mixin atomic-grid-with-cards {
-    @mixin atomic-list-with-cards;
-    grid-column-gap: 0.5rem;
-    grid-row-gap: 0.5rem;
-  }
-
-  @screen mobile-only {
-    &.image-large {
-      @media not all and (min-width: 768px) {
-        @mixin atomic-list-with-dividers;
-        grid-template-columns: minmax(auto, 35rem);
-      }
-      @media (min-width: 768px) {
-        @mixin atomic-grid-with-cards;
-        grid-template-columns: 1fr 1fr;
-      }
-    }
-    &.image-small,
-    &.image-icon,
-    &.image-none {
-      @mixin atomic-grid-with-cards;
-      @media not all and (min-width: 640px) {
-        grid-template-columns: minmax(0, 1fr);
-      }
-      @media (min-width: 768px) {
-        grid-template-columns: minmax(0, 1fr) minmax(0, 1fr);
-      }
-      @media (min-width: 1024px) {
-        grid-template-columns: minmax(0, 1fr) minmax(0, 1fr) minmax(0, 1fr);
-      }
-    }
-  }
-}
+@import './styles/placeholders.pcss';
```

Comment:
I didn't change any of the existing CSS, I simply split it & included only the necessary for every result list